src/main-process/atom-window.js

```diff
@@ -485,13 +485,13 @@ module.exports = class AtomWindow extends EventEmitter {
       options
     );
 
+    let promise =  dialog.showSaveDialog(this.browserWindow, options);
```

Comment:
This line will probably fail linting. It needs this fix: https://github.com/atom/atom/pull/21935/commits/2f0711425d321955aa17c11a665e3089bd1e8b42

(@asturur did that for the other PR where the tests were running. But it was less obvious here.)